nutkit/protocol/feature.py

```diff
@@ -13,6 +13,9 @@ class Feature(Enum):
     # is picked up instead or we need to wait until the full pool depletes.
     API_CONNECTION_ACQUISITION_TIMEOUT = \
         "Feature:API:ConnectionAcquisitionTimeout"
+    # The driver offers a method for run a query in a retriable context at the
```

Comment:
```suggestion
    # The driver offers a method for run a query in a retryable context at the
```
I vote for retryable (also in test names, etc.) As that's the spelling more commonly used throughout TestKit.